types/mydecimal.go

```diff
@@ -107,6 +107,14 @@ var (
 	zeroMyDecimal = MyDecimal{}
 )
 
+// get the zero of MyDecimal with the specified result fraction digits
+func zeroMyDecimalWithFrac(frac int8) MyDecimal {
```

Comment:
How about generating a list of zero MyDecimals at the beginning so that we don't need to generate a MyDecimal each time.

Reply:
I don't think it's a good idea.
What if `select 0.0000000000000000000000000000000000000000000000000000000000000000000000 % 0.1;` (70 zero after point)?